pkg/utils/keypath/safe_point.go

```diff
@@ -18,19 +18,74 @@ import (
 	"fmt"
 )
 
-// GCSafePointPath returns the GC safe point key path.
+// GCStateRevisionPath returns the key path for storing the revision of GC state data.
+func GCStateRevisionPath() string {
+	return fmt.Sprintf(gcStateRevisionPathFormat, ClusterID())
+}
+
+// GCSafePointPath returns the key path of the global (NullKeyspace) GC safe point.
```

Comment:
How about explicitly introducing and explaining the relationship between global and NullKeyspace in the code comments? This can help everyone understand its meaning clearly.